tracee/tracee.go

```diff
@@ -407,8 +441,9 @@ func (t *Tracee) processFileWrites() {
 				continue
 			}
 
-			pathname := path.Join(t.config.OutputPath, strconv.Itoa(int(meta.MntID)))
-			if err := os.MkdirAll(pathname, 0755); err != nil {
+			pathname := path.Join(t.config.OutputPath, strconv.Itoa(int(meta.MntID)), "written-files")
```

Comment:
I'm don't think we need this extra hierarchy now ("written-files").
File names are different for vfs_writes (starts with "write.") and unpacker (not yet in PR, starts with "bin."), and executed files can start with "exe." prefix.
Do you see a resaon for the extra path here? we can always add it in the future

Reply:
I don't see a reason for hierarchy but I also don't see the problem. I don't know if adding the dir name to the file name is better.. but I don't have strong feelings about this so you prefer it this way I can change